### PR TITLE
LOK-2342 Document the properties in values files

### PR DESCRIPTION
**Pull request by Umairk95**

### CODE REVIEW

File: dependencies/citus/values.yaml [hunk 1]
```diff
@@ -6,19 +6,22 @@ OpenNMS: # we might be able to remove this
   
 citus:
   enabled: true
-  workersEnabled: true
+  workersEnabled: true
```
Comment (mershad-manesh) [inline dependencies/citus/values.yaml:9]: I think we can remove the extra space at the end of this line

File: dependencies/cortex/values.yaml [hunk 2]
```diff
@@ -4,13 +4,13 @@ OpenNMS: # we might be able to remove this
...
-      otlpTracesEndpoint: null
-      env:
+      otlpTracesEndpoint: null # Specifies the OTLP traces endpoint URL
+      env: # Environment variable for OpenTelemetry used in Spring-boot-env-configmap
         OTEL_PROPAGATORS: tracecontext,baggage,jaeger
 cortex:
   enabled: true
   serviceName: cortex
-  port: 9000
+  port: 9000 # ConfigMap http port
```
Comment (mershad-manesh) [inline dependencies/cortex/values.yaml:13]: I feel like we can expand on what we mean by `ConfigMap http port`. 
Maybe we can just say what this port number is used for.

File: dependencies/mail-server/values.yaml [hunk 3]
```diff
@@ -8,13 +8,13 @@ OpenNMS: # we might be able to remove this
...
 mailServer:
-  enabled: false
+  enabled: false # to disable mail server change it to false
   serviceName: mail-server
   image: mailhog/mailhog:v1.0.1
   imagePullPolicy: IfNotPresent
   replicas: 1
   port: 8025
-  smtpPort: 1025
+  smtpPort: 1025 #smtp port
```
Comment (mershad-manesh) [inline dependencies/mail-server/values.yaml:17]: I think we can remove this comment

File: lokahi-minion/values.yaml [hunk 4]
```diff
@@ -5,26 +5,26 @@ global:
...
+      env: # Environment variable for OpenTelemetry used in Spring-boot-env-configmap
+        OTEL_PROPAGATORS: tracecontext,baggage,jaeger
   developmentMode: false
 minion:
   # Set an ID for Minion
   id: ""
   # Enable this option if you are placing a `ca.cert` under the `certs` folder
   customCertificateAuthority: false
-  grpcClientKeystorePassword: ""
+  grpcClientKeystorePassword: ""  # GRPC_CLIENT_KEYSTORE_PASSWORD
```
Comment (mershad-manesh) [inline lokahi-minion/values.yaml:17]: We should expand on what `GRPC_CLIENT_KEYSTORE_PASSWORD` is

**Review by mershad-manesh — CHANGES_REQUESTED** [review]
Overall it looks good. A couple of items needs to be tweaked

File: dependencies/citus/values.yaml (hunk 1 above)
Comment (Umairk95) [reply dependencies/citus/values.yaml]: made changes

File: lokahi-minion/values.yaml (hunk 4 above)
Comment (Umairk95) [reply lokahi-minion/values.yaml]: changes done

**Review by MuhammadTalhaDataQHealth — APPROVED** [review]

**Pull request merged**